web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FullRecordController.java

```diff
@@ -370,6 +349,33 @@ public void setXslViewResolver(XSLViewResolver xslViewResolver) {
         this.xslViewResolver = xslViewResolver;
     }
 
+    private HashMap<String, Object> getViewerProperties(ContentObjectRecord briefObject, AccessGroupSet principals) {
```

Comment:
Sorry to be pedantic, but when declaring return types we should prefer the interface over the implementation, unless the implementation is important. Keeps things more flexible.